src/components/TeamGridLayout/index.tsx

```diff
@@ -1,63 +1,48 @@
+import React, { useState } from "react";
```

Comment:
React is never used, which will throw a build error from Render

Reply:
fixed